std/os/darwin.zig

```diff
@@ -5,3 +5,4 @@ pub const is_the_target = switch (builtin.os) {
     else => false,
 };
 pub usingnamespace std.c;
+pub usingnamespace @import("bits.zig");
```

Comment:
why is this added?

Reply:
This was copied from the Linux file, as `S_ISCHR` was coming in from the `bits`: https://github.com/ziglang/zig/blob/master/std/os/linux.zig#L22 - I also added it to `os/freebsd.zig`. It can be removed from both.

Reply:
Oh, hmm. Yes I think what you have here is correct. Thanks!